src/core/unit.c

```diff
@@ -721,6 +721,12 @@ void unit_free(Unit *u) {
         free(u);
 }
 
+FreezerState freezer_state(Unit *u) {
```

Comment:
`unit_freezer_state`, judging by convention for other state retrievals that take a unit (like unit_sub_state)